config/schemas/config.yaml

```diff
@@ -208,13 +316,29 @@ $defs:
       type: object
     type: array
   kubernetesTopologySpreadConstraints:
+    title: Kubernetes Topology Spread Constraints Config
```

Comment:
I'm not sure if we need to suffix all titles with `Config`, I've mainly done it where it is either part of something bigger or to make it clear when the same key exists in as secrets as well. :thinking:

Reply:
Yeah, good pint. I clean it up :+1: